source/_integrations/homewizard.markdown

```diff
@@ -53,50 +53,70 @@ IP address:
 
-- **Voltage (V)**: Active voltage measured on each phase.
-- **Current (A)**: Active current measured on each phase.
-- **Frequency (Hz)**: Net frequency.
-- **Reactive power (VAR)**: Active reactive power measurement on each phase.
-- **Apparent power (VA)**: Active apparent power measurement on each phase.
-
-Sensors for Water meter:
+### kWh Meter
 
-- **Water usage (L/min)**: Flow of water measured at that time.
-- **Total water usage (m³)**: Total water usage since the installation of the HomeWizard Water meter.
+- **Energy import/export (kWh)**: Total energy imported or exported measured by kWh meter.
+- **Power (W)**: Active power that is measured, each phase has its own sensor.
+- **Voltage (V)**: Active voltage measured, each phase has its own sensor.
+- **Current (A)**: Active current measured, each phase has its own sensor.
+- **Frequency (Hz)**: Net frequency.
+- **Reactive power (VAR)**: Active reactive power measurement, each phase has its own sensor.
+- **Apparent power (VA)**: Active apparent power measurement, each phase has its own sensor.
 
-Sensors for Plug-In Battery:
+### Energy Socket
 
-- **Cycles**: Number of charge cycles the battery has gone through.
-- **State of charge (%)**: The current state of charge of the battery.
+- **Energy import/export (kWh)**: Total energy imported or exported measured by Energy Socket.
+- **Power (W)**: Active power that is measured.
+- **Voltage (V)**: Active voltage measured.
+- **Current (A)**: Active current measured.
+- **Frequency (Hz)**: Net frequency.
+- **Reactive power (VAR)***: Active reactive power measurement.
+- **Apparent power (VA)***: Active apparent power measurement.
 
-## Energy Socket
+{% note %}
+Reactive power and apparent power are not available for all hardware versions, this depends on the internal metering chip.
+{% endnote %}
 
-The Energy Socket outlet state and status light can be controlled. There are two switches:
+The Energy Socket also has a switch to control the outlet state and a status light that can be controlled.
 
 - **Switch**: Controls the outlet state of the Energy Socket. This switch is permanently on when _Switch Lock_ is turned on. Use this to control the power of simple devices, such as a heater or a charger.
 - **Switch lock**: Forces the outlet state to the _on_ position and disables the physical button. This option is useful when the socket is used for a device that must not be turned off, such as a refrigerator.
+- **Status light brightness**: Controls the brightness of the green status light. This light turns on when the switch is on.
 
-You can also control the green status light brightness with **Status light brightness**. This light turns on when the switch is on.
+### Watermeter
+
+- **Water usage (L/min)**: Flow of water measured at that time.
+- **Total water usage (m³)**: Total water usage since the installation of the HomeWizard Water meter.
```

Comment:
```suggestion
- **Total water usage (m³)**: Total water usage since the installation of the HomeWizard water meter.
```

Reply:
```suggestion
- **Total water usage (m³)**: Total water usage since the installation of the HomeWizard Watermeter.
```

Same as Plug-In Battery, it is branding so I prefer to keep it like this. 'Water meter' is indeed incorrect.